src/lib/sfge/AbstractSfgeEngine.ts

```diff
@@ -164,9 +166,23 @@ export abstract class AbstractSfgeEngine extends AbstractRuleEngine {
 	 */
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	public shouldEngineRun(ruleGroups: RuleGroup[], rules: Rule[], target: RuleTarget[], engineOptions: Map<string,string>): boolean {
-		// If the engine wasn't filtered out, there's no reason not to run it.
-		// TODO: WE MAY WANT TO RE-ASSESS THIS.
-		return true;
+		// By now, the thing to check is whether we have the necessary engineOptions information to actually
```

Comment:
IMPORTANT CHANGE: If the config is missing, then `shouldEngineRun` returns false. Doing anything else would introduce many test failures, dramatically inflating the size of the PR. As noted by the TODO, we could fix this in the upcoming story about post-install scripts.

Reply:
Makes sense 👍